examples/error-user-defined/error_user_defined.bal

```diff
@@ -0,0 +1,107 @@
+import ballerina/io;
```

Comment:
It is recommended to limit the maximum character count per line to 80 in .bal files. Otherwise, the lines get automatically wrapped in the Ballerina By Example reducing the readability. So please check all the file and break the line by 80 chars. 

[1] https://github.com/ballerina-platform/ballerina-lang/blob/master/docs/bbe-guidelines/bbe-guidelines.md